src/pages/index.html

```diff
@@ -6,11 +6,18 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
   <link rel="stylesheet" href="../../assets/css/output.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css"
+    integrity="sha512-Kc323vGBEqzTmouAECnVceyQqyqdsSiqLQISBL29aUW4U/M7pSPA/gEUZQqv1cwx4OnYxTxve5UMg5GT6L4JJg=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link href="https://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+  <link href="https://fonts.cdnfonts.com/css/open-sans" rel="stylesheet">
 </head>
 
 <body>
-  <h1 class="text-3xl font-bold underline text-center text-red-900">
-    Hello world!
-  </h1>
+  <div id="header"></div>
+  <div id="footer"></div>
+
+  <script src="../../assets/js/i18n.js"></script>
+  <script src="../../assets/js/loadlayouts.js" type="module"></script>
```

Comment:
```suggestion
  <script src="../../assets/js/loadlayouts.js" type="module"></script>
  <script src="../../assets/js/i18n.js"></script>
```
E đổi thứ tự lại để i18n chạy sau cùng nha